anywayapp/models.py

```diff
@@ -0,0 +1,1938 @@
+# This is an auto-generated Django model module.
+# You'll have to do the following manually to clean this up:
+#   * Rearrange models' order
+#   * Make sure each model has one field with primary_key=True
+#   * Make sure each ForeignKey and OneToOneField has `on_delete` set to the desired behavior
+#   * Remove `managed = False` lines if you wish to allow Django to create, modify, and delete the table
+# Feel free to rename the models, but don't rename db_table values or field names.
+from django.contrib.gis.db import models
+
+
+class AccidentHourRaw(models.Model):
+    id = models.IntegerField(primary_key=True)
```

Comment:
Just a thought, in case we won't define a primary key, django will do it on it's own, do we want to define it by ourselves?

Reply:
same goes for the rest of the models of course